src/admin/datatypes/index.md

```diff
@@ -3,6 +3,5 @@
 Hub page for [administering](/src/admin/index.md) Galaxy data types.
 
 * [Admin/ReferenceMAFs](/src/admin/reference-mafs/index.md)
-* [Adding Completely new Datatypes](/src/admin/datatypes/Adding Complete Datatypes/index.md)
```

Comment:
it was not even in the old wiki: https://moin.galaxyproject.org/Admin/Datatypes and the link was broken. Will add it back.